legacy-usage.md

```diff
@@ -0,0 +1,448 @@
+
```

Comment:
```suggestion
# Using this library in your project

```

Also consider adjusting the headers in this doc, so that they start from a single `#` and properly increase by one char per nesting.

Reply:
Yeah, it was just copy/paste.